### ISSUE TITLE
Setuptools 72 incompatible with this library

### ISSUE THREAD

**Issue by fwfichtner** [ezhov-evgeny/webdav-client-python-3#147]
https://github.com/pypa/setuptools/issues/4519 breaks the installation of this package

**Comment by alex-pope-l3** [comment]
We've been having the same problem.  Pinning `setuptools<72.0.0` in `pyproject.toml` for the effected package worked for us.

**Comment by fwfichtner** [comment]
Yes, we had set a `PIP_CONSTRAINT`, but I guess these are just workarounds and should best be fixed in here. Maybe this could be combined with https://github.com/ezhov-evgeny/webdav-client-python-3/issues/144? 

Is anyone still maintaining this?

**Comment by alex-pope-l3** [comment]
Agreed. Pinning setuptools to never upgrade to version 72 or above is a temporary workaround and not a sustainable long term solution. The fix should definitely reside in here and probably should be combined with #144.

As the last commit to this repo was more than a year ago, we are operating under the assumption that this package isn't currently being maintained, though whether that is a permanent state of affairs is unclear, and plan to start evaluating other WebDAV libraries.

**Comment by fwfichtner** [comment]
I changed the library to https://github.com/skshetry/webdav4

**Comment by michael-o** [comment]
> I changed the library to https://github.com/skshetry/webdav4

Unfortunately, this one uses httpx and not requests. Means that it also changes the backend config, etc.
